vibi-dpu/src/github/webhook.rs

```diff
@@ -73,21 +72,20 @@ async fn process_add_webhook_response(response: Result<Response, Error>){
             res.status(), res.text().await);
         return;
     }
-    let webhook_res = res.json::<Webhook>().await;
-    if webhook_res.is_err() {
-        let err = webhook_res.expect_err("No error in webhook response");
-        eprintln!("Failed to parse webhook_res: {:?}", err);
-        return;
-    }
-    let webhook = webhook_res.expect("Uncaught error in webhook response");
-    let webhook_data = Webhook::new(
-        webhook.id().to_string(),
-        webhook.active(),
-        webhook.created_at().to_string(),
-        webhook.events().to_owned(),
-        webhook.ping_url().clone(),
-        webhook.url().to_string(),
-        webhook.config().clone()
+    let webhook_json = res.json::<Value>().await.expect("[process_add_webhook_response] Unable to deserialize res to Value");
+    let webhook = Webhook::new(
+        webhook_json["id"].to_string(),
+        webhook_json["active"].as_bool().expect("Unable to deserialize active"),
+        webhook_json["created_at"].to_string().replace('"', ""),
+        webhook_json["events"].as_array().expect("Unable to deserialize events").into_iter()
+            .map(|events| events.as_str().expect("Unable to convert event").to_string()).collect(),
+        webhook_json["ping_url"].to_string().replace('"', ""),
+        webhook_json["config"]["url"].to_string().replace('"', ""),
+        webhook_json.get("config")
```

Comment:
I see only 6 parameters to the constructor function of Webhook. I couldn't understand the existence of this 7th argument.